MatrixSDK/Background/Store/SyncResponseFileStore/SyncResponseFileStore.swift

```diff
@@ -128,6 +128,38 @@ extension SyncResponseFileStore: SyncResponseStore {
         return result
     }
     
+    public func roomSummary(forRoomId roomId: String) -> MXRoomSummary? {
+        guard let response = syncResponse else {
+            return nil
+        }
+        if let invitedRoomSync = response.rooms.invite[roomId],
```

Comment:
Note that we need to do the same for joined rooms. But I think this is too hard to implement now.

The use case is the user who has 2 devices. They join a room on deviceA and starts receiving pushes in it. They never started deviceB. DeviceB has no data in its MXFileStore about this room. It will not be able to display a good notification.